### ISSUE TITLE
Unknown error, cannot send

### ISSUE THREAD

**Issue by KohnoseLami** [interactions-py/molter#5]
After the first command execution, it will be sent, but after the second execution, it will not be sent.
```py
@molt.prefixed_command()
async def backup(ctx):
    if not ctx.guild is None:
        if ctx.author.permissions & interactions.Permissions.ADMINISTRATOR:
            await ctx.defer()
            await ctx.get_guild()
            members = await ctx.guild.get_all_members()
            roles = await ctx.guild.get_all_roles()
            channels = await ctx.guild.get_all_channels()
            test = {"members": [{"id": int(member.id), "nick": member.nick, "roles": member.roles} for member in members], "channels": [{"id": int(channel.id), "name": channel.name, "nsfw": channel.nsfw, "parent_id": channel.parent_id, "permission_overwrites": channel.permission_overwrites, "type": channel.type, "position": channel.position} for channel in channels], "roles": [{"id":int(role.id), "name": role.name, "color": role.color, "permissions": int(role.permissions), "position": role.position} for role in roles]}
            for i, _ in enumerate(test["channels"]):
                if _["parent_id"]:
                    test["channels"][i]["parent_id"] = int(_["parent_id"])
                if _["permission_overwrites"]:
                    test["channels"][i]["permission_overwrites"] = [{"id": int(i.id), "type": int(i.type), "allow": int(i.allow), "deny": int(i.deny)} for i in _["permission_overwrites"]]
            file = io.StringIO(json.dumps(test, indent=4, ensure_ascii=False))
            with file as f:
                file = interactions.File(filename="test.json", fp=f)
                await ctx.send(files=file)
            return
            #backup_dict[str(ctx.guild.id)] = {"members": [{"id": int(member.id), "nick": member.nick, "roles": [role.name for role in member.roles]} for member in members], "channels": [{"name": channel.name, "type": channel.type[1], "position": channel.position} for channel in channels], "roles": [{"name": role.name, "position": role.position} for role in roles]}
            embed = discord.Embed(title="Backup", description=f"{ctx.guild.name} をバックアップしました", color=0xff0000)
            await ctx.send(embeds=[interactions.Embed(**embed.to_dict())])
        else:
            embed = discord.Embed(title="Backup", description="管理者ではないためコマンドを実行することはできません", color=0xff0000)
            await ctx.send(embeds=[interactions.Embed(**embed.to_dict())])
    else:
        embed = discord.Embed(title="Backup", description="DMでは使用できません", color=0xff0000)
        await ctx.send(embeds=[interactions.Embed(**embed.to_dict())])
```

```py
ERROR:molter:Ignoring exception in molter cmd / :
Traceback (most recent call last):
  File "C:\Users\ikki1\AppData\Local\Programs\Python\Python39\lib\site-packages\interactions\ext\molter\base.py", line 394, in _handle_prefixed_commands
    await command(context)
  File "C:\Users\ikki1\AppData\Local\Programs\Python\Python39\lib\site-packages\interactions\ext\molter\command.py", line 710, in __call__
    return await self.invoke(ctx)
  File "C:\Users\ikki1\AppData\Local\Programs\Python\Python39\lib\site-packages\interactions\ext\molter\command.py", line 722, in invoke
    return await callback(ctx)
  File "D:\Desktop\Projects\NX\main.py", line 1299, in backup
    await ctx.send(embeds=[interactions.Embed(**embed.to_dict())])
  File "C:\Users\ikki1\AppData\Local\Programs\Python\Python39\lib\site-packages\interactions\ext\molter\context.py", line 172, in send
    channel = await self._get_channel_for_send()
  File "C:\Users\ikki1\AppData\Local\Programs\Python\Python39\lib\site-packages\interactions\ext\molter\context.py", line 119, in _get_channel_for_send
    return channel.value
AttributeError: 'Channel' object has no attribute 'value'
```

**Comment by Astrea49** [comment]
Odd. Seems like `interactions.py`'s cache forgets to put things in `Item`s sometimes?

This is a decently simple fix, though... and it's by avoiding the cache altogether 😅 - I'll have a fix out soon.

**Comment by KohnoseLami** [comment]
> 

I see. Taking the hint that it is not stored in the cache as a hint, I executed "await ctx.get_channel()" and sent, and it worked properly.
```py
@molt.prefixed_command()
async def backup(ctx):
    await ctx.get_channel()
    if ctx.guild_id:
        print(dir(ctx.author))
        if ctx.author.permissions & interactions.Permissions.ADMINISTRATOR:
            await ctx.defer()
            await ctx.get_guild()
            members = await ctx.guild.get_all_members()
            roles = await ctx.guild.get_all_roles()
            channels = await ctx.guild.get_all_channels()
            guild = {"users": [int(ctx.user.id)], "members": [{"id": int(member.id), "nick": member.nick, "roles": member.roles} for member in members], "channels": [{"id": int(channel.id), "name": channel.name, "nsfw": channel.nsfw, "parent_id": channel.parent_id, "permission_overwrites": channel.permission_overwrites, "type": channel.type, "position": channel.position} for channel in channels], "roles": [{"id":int(role.id), "name": role.name, "color": role.color, "permissions": int(role.permissions), "position": role.position} for role in roles]}
            for i, _ in enumerate(guild["channels"]):
                if _["parent_id"]:
                    guild["channels"][i]["parent_id"] = int(_["parent_id"])
                if _["permission_overwrites"]:
                    guild["channels"][i]["permission_overwrites"] = [{"id": int(i.id), "type": int(i.type), "allow": int(i.allow), "deny": int(i.deny)} for i in _["permission_overwrites"]]
            backup_dict.put(guild, str(ctx.guild_id))
            embed = discord.Embed(title="Backup", description=f"{ctx.guild.name} をバックアップしました", color=0x33b2b8)
            await ctx.send(embeds=[interactions.Embed(**embed.to_dict())])
        else:
            embed = discord.Embed(title="Backup", description="管理者ではないためコマンドを実行することはできません", color=0xff0000)
            await ctx.send(embeds=[interactions.Embed(**embed.to_dict())])
    else:
        embed = discord.Embed(title="Backup", description="DMでは使用できません", color=0xff0000)
        await ctx.send(embeds=[interactions.Embed(**embed.to_dict())])
```

For some reason, the first command seems to be stored, but the second time, the channel doesn't seem to exist?

**Comment by Astrea49** [comment]
Yeah, I'm not sure what's entirely going on, haha... this feels like a problem that's beyond `molter`, somehow.

Regardless, 4c85e31bac21b73b1b4f24e7189f313e46075735 *should* fix it, and a bug fix release will come out soon!

**Comment by Astrea49** [comment]
Fixed on version `0.4.3`! Let me know if that still has problems, but it shouldn't, hopefully.

**Comment by KohnoseLami** [comment]
Thank you.
Confirmed fix
